docs/ops.html

```diff
@@ -4169,6 +4170,7 @@ <h4 class="anchor-heading"><a id="tiered_storage_limitation" class="anchor-link"
   <li>Deleting tiered storage enabled topics is required before disabling tiered storage at the broker level</li>
```

Comment:
Please also reword this sentence:

Deleting tiered storage enabled topics  ... -> Disabling tiered storage enabled topics ...

Reply:
done